app/theme/client/main.css

```diff
@@ -58,3 +58,417 @@
 
 /* RTL */
 @import 'imports/general/rtl.css';
+
+/* customerChat History CSS */
+.inner-addon {
+	position: relative;
+        left: 24px;
+    
+	    width: 345px;
+        height: 40px;
+    
+    border: 2px solid #cbced1;
+
+    box-sizing: border-box;
+    background: #ffffff;
+
+	border-radius: 2px;
+}
+
+.inner-addon .glyphicon {
+	position: absolute;
+	    bottom: 0.5%;
+    color: #2F343D;
+    
+	padding: 10px;
+    
+    pointer-events: none;
+	
+}
+
+.searchresults {
+    margin-top: 7%;
+}
+
+.search-li {
+    display: block;
+    padding-bottom: 3%;
+    position: relative;
+
+    height: 65px;
+    
+    background-color: #ffffff;
+}
+
+.search-li :hover { 
+     cursor: default;
+}
+
+.closed-div {
+    position: relative;
+        top: 20px;
+}
+
+.closed-div-p {
+        background: #f5455c;
+    border-radius: 2px;
+    width: 100%;
+    text-align: center;
+    line-height: 0.15em;
+    font-size: 10px;
+        
+        margin: 10px 0 15px;
+}
+
+.closed-div-p span {
+        width: 101px;
+    height: 12px;
+        
+        right: 138px;
+    top: 3px;
+    font-family: Inter;
+    font-style: normal;
+    font-weight: 600;
+    text-align: center;
+        
+        color: #f5455c;
+    background: white;
+    padding-left: 8px;
+    padding-right: 8px;
+}
+
+.closed-h1 h1 {
+        color: #6c727a !important;
+        
+        font-size: 14px !important;
+        
+        line-height: 20px !important;
+        
+    font-weight: 600px !important;
+    font-style: italic !important;
+    text-transform: none !important;
+}
+
+.closed-p p {
+        color: #9ea2a8 !important;
+    font-style: italic !important;
+    font-weight: 500 !important;
+    top: 25px !important;
+}
+
+.belldiv {
+    position: absolute;
+    width: 36px;
+    height: 36px;
+    top: 12px;
+    left: 24px;
+    background: #e4e7ea;
+}
+
+.belldiv i {
+    position: relative;
+    left: 22.31%;
+    top: 15.79%;
+}
+
+.history-main {
+    display: flex;
+    position: relative;
+        
+        width: 100%;
+    height: 68px;
+    left: 2px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5378317)